docs/ci.md

```diff
@@ -20,6 +38,34 @@ run on normal runners.
 4. Enter a branch name or a PR number in the input field.
 5. Click the green `Run workflow` button.
 
-### Trigger via PR Label
+Here is an example of using the GitHub Web UI to launch an E2E workflow:
+
+![GitHub Actions Run Workflow Example](images/github-actions-run-workflow-example.png)
+
+### Current GPU-enabled Runners
+
+The project currently supports the usage of the following runners for the E2E jobs:
+
+* [GitHub built-in GPU runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners#specifications-for-gpu-larger-runners) -- referred to as `ubuntu-gpu` in our workflow files Only `e2e.yml` uses this runner.
+* Ephemeral GitHub runners launched on demand on AWS. Most workflows work this way, granting us access to a wider variety of infrastructure at lower cost.
+
+### E2E Workflows
+
+| File | Runner Host |Instance Type | GPU Type | OS |
+| --- | --- | --- | --- | --- |
+| [`e2e.yml`](https://github.com/instructlab/instructlab/blob/main/.github/workflows/e2e.yml) | GitHub | N/A | 1 x NVIDIA Tesla T4 w/ 16 GB VRAM | Ubuntu |
+| [`e2e-nvidia-t4-x1.yml`](https://github.com/instructlab/instructlab/blob/main/.github/workflows/e2e-nvidia-t4-x1.yml) | AWS | [`g4dn.2xlarge`](https://aws.amazon.com/ec2/instance-types/g4/) | 1 x NVIDIA Tesla T4 w/ 16 GB VRAM | CentOS Stream 9 |
```

Comment:
I wonder if we might want to deprecate the `e2e.yml` job in favor of the `e2e-nvidia-t4-x1.yml` job since it seems to be the same specs but a lower cost

Reply:
Yes, absolutely.  Two things

1. I wanted to make sure it was stable, thought these jobs seem very reliable so far.
2. Since this job uses secrets (the AWS and github keys), it's not as simple as just changing the workflow to be triggered on PRs. GitHub won't allow it for security reasons. There are at least a couple of options to consider ..